pkg/clientcache/cache.go

```diff
@@ -85,24 +85,24 @@ func (c *ClientCache) CreateClient(contextName string, restConfig *rest.Config)
 }
 
 // CreateRemoteClientsFromSecret is a convenience method for testing purposes
-func (c *ClientCache) CreateRemoteClientsFromSecret(namespacedName types.NamespacedName) error {
+func (c *ClientCache) CreateRemoteClientsFromSecret(secretKey types.NamespacedName) error {
```

Comment:
This method doesn't look in use right now.

Reply:
Yeah, I think it can be removed.